exercises/parallel-letter-frequency/tests/parallel-letter-frequency.rs

```diff
@@ -3,37 +3,40 @@ use std::collections::HashMap;
 extern crate parallel_letter_frequency as frequency;
 
 // Poem by Friedrich Schiller. The corresponding music is the European Anthem.
-const ODE_AN_DIE_FREUDE: [&'static str; 8] = [
-    "Freude schöner Götterfunken",
-    "Tochter aus Elysium,",
-    "Wir betreten feuertrunken,",
-    "Himmlische, dein Heiligtum!",
-    "Deine Zauber binden wieder",
-    "Was die Mode streng geteilt;",
-    "Alle Menschen werden Brüder,",
-    "Wo dein sanfter Flügel weilt."];
+const ODE_AN_DIE_FREUDE: [&'static str; 8] = ["Freude schöner Götterfunken",
+                                              "Tochter aus Elysium,",
+                                              "Wir betreten feuertrunken,",
+                                              "Himmlische, dein Heiligtum!",
+                                              "Deine Zauber binden wieder",
+                                              "Was die Mode streng geteilt;",
+                                              "Alle Menschen werden Brüder,",
+                                              "Wo dein sanfter Flügel weilt."];
 
 // Dutch national anthem
-const WILHELMUS: [&'static str; 8] = [
-    "Wilhelmus van Nassouwe",
-    "ben ik, van Duitsen bloed,",
-    "den vaderland getrouwe",
-    "blijf ik tot in den dood.",
-    "Een Prinse van Oranje",
-    "ben ik, vrij, onverveerd,",
-    "den Koning van Hispanje",
-    "heb ik altijd geëerd."];
+const WILHELMUS: [&'static str; 8] = ["Wilhelmus van Nassouwe",
+                                      "ben ik, van Duitsen bloed,",
+                                      "den vaderland getrouwe",
+                                      "blijf ik tot in den dood.",
+                                      "Een Prinse van Oranje",
+                                      "ben ik, vrij, onverveerd,",
+                                      "den Koning van Hispanje",
+                                      "heb ik altijd geëerd."];
 
 // American national anthem
-const STAR_SPANGLED_BANNER: [&'static str; 8] = [
-    "O say can you see by the dawn's early light,",
-    "What so proudly we hailed at the twilight's last gleaming,",
-    "Whose broad stripes and bright stars through the perilous fight,",
-    "O'er the ramparts we watched, were so gallantly streaming?",
-    "And the rockets' red glare, the bombs bursting in air,",
-    "Gave proof through the night that our flag was still there;",
-    "O say does that star-spangled banner yet wave,",
-    "O'er the land of the free and the home of the brave?"];
+const STAR_SPANGLED_BANNER: [&'static str; 8] = ["O say can you see by the dawn's early light,",
```

Comment:
this one's a bit unfortunate, I am sure I liked the previous style better, but then again I don't really think fighting rustfmt is a thing I want to do